### PR TITLE
chore: add safeareaview to create alert button

### PR DESCRIPTION
**Pull request by gkartalis**
The type of this PR is: **Enhancement**

### Description

Adds SafeAreaView in order to provide better ux to users with iPhones. 
Before this improvements users with iPhones without physicals buttons had a hard time pressing the create alert buttons since it was too close to the 

| Before | After |
|---|---|
|Simulator Screen Shot - iPhone 12 - 2022-02-02 at 15 35 32|Simulator Screen Shot - iPhone 12 - 2022-02-02 at 15 35 15|

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

- adds safeareaview to create alert button - gkartalis

#### Android user-facing changes

-

#### Dev changes

-

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- iOS user-facing changes (adds safeareaview to create alert button - gkartalis)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 736851984f8c34a523edc34b2254638386077a4a
</p>

File: src/lib/Components/ArtworkFilter/components/ArtworkFilterApplyButton.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useFeatureFlag } from "lib/store/GlobalStore"
 import { BellIcon, Box, Button, Separator, Text, useColor } from "palette"
 import React, { ReactNode, useState } from "react"
-import { Pressable } from "react-native"
+import { Pressable, SafeAreaView } from "react-native"
```
Comment (dimatretyak) [inline src/lib/Components/ArtworkFilter/components/ArtworkFilterApplyButton.tsx:4]: **suggestion**: I think it's better to use `SafeAreaView` from `react-native-safe-area-context`

**Comment by dimatretyak** [comment]
Good catch. Thanks for the fixes!

**Review by mdole — APPROVED** [review]
+1 to Dima's suggestion, otherwise LGTM!!

**Comment by gkartalis** [comment]
closing to find a better solution cause this doesn't work for iphone X and below. Will bring it up to design to see if we want to keep the solution from other screens (bottom padding 30) or another one

**Pull request closed without merging**